src/main/java/io/odpf/firehose/metrics/Instrumentation.java

```diff
@@ -192,6 +192,10 @@ public void captureDurationSince(String metric, Instant instant) {
         statsDReporter.captureDurationSince(metric, instant);
     }
 
+    public void captureDurationSinceWithTags(String metric, Instant instant, String... tags) {
```

Comment:
we can add varargs to captureDurationSince method itself instead of creating a new method

Reply:
yes. I did this in the other PR :D